src/assets/scripts/game/GameClockView.js

```diff
@@ -0,0 +1,96 @@
+import $ from 'jquery';
+import { digits_integer } from '../utilities/radioUtilities';
+import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * @class GameClockView
+ */
+export default class GameClockView {
```

Comment:
though this class is somewhat self explanatory, be sure to add a quick blurb about what this class is doing